content/docs/command-reference/import.md

```diff
@@ -276,3 +278,108 @@ outs:
 
 The `url` and `rev_lock` subfields under `repo` are used to save the origin and
 [version](https://git-scm.com/docs/revisions) of the dependency, respectively.
+
+## Example: Chained imports
+
+DVC supports importing data that was itself imported into the source repo, as
+long as all the repos in the import chain (and their default remote storage) are
+accessible from the final destination repo.
+
+Consider an example with 3 DVC repos (A, B, and C). DVC repo `/repo/a` contains
+a `data.csv` file tracked with `dvc add`:
+
+```
+/repo/a
+├── data.csv
+└── data.csv.dvc
+```
+
+In repo B, we import `data.csv` from A and into a subdirectory:
+
+```dvc
+$ dvc import /repo/a data.csv --out training/data.csv
+```
+
+The project may of course have other files, for example:
+
+```
+/repo/b
+└── training
+    ├── data.csv
+    ├── data.csv.dvc
+    ├── labels
+    │   ├── test.txt
+    │   └── truth.txt
+    └── labels.dvc
+```
+
+> Notice that the `training/labels` directory is also tracked by DVC separately.
```

Comment:
@jorgeorpinel This is where the removed `outs` section came from (it's the contents of `labels.dvc`). It may be better to move this below the `training/data.csv.dvc` block so it says something like:

If we also examine `training/labels.dvc` we can see that it is a directory tracked by DVC and not an import. Meaning that `training/` contains a both data unique to repo B and imports from repo A.
```
outs:
- md5: 630bd47b538d2a513c7d267d07e0bc44.dir
  size: 3
  nfiles: 1
  path: labels
```

Reply:
Got it. Updated to use some of your language in 4bf55012. Thanks!